release notes/v1.2.0.md

```diff
@@ -0,0 +1,363 @@
+k6 v1.2.0 is here 🎉! This release includes:
+
+- (_optional_) `<highlight of breaking changes>`
+- `<Summary of new features>` (_one or multiple bullets_)
+
+## Breaking changes
```

Comment:
https://github.com/grafana/k6/releases/tag/v1.1.0 I would like to replicate the same section as we did the last time.